### PR TITLE
Bump all misc, non-product versions

### PR DESCRIPTION
**Pull request by sbernauer**
# Description

Part of https://github.com/stackabletech/issues/issues/594

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes
 
```[tasklist]
- [ ] Changes are OpenShift compatible
- [ ] All added packages (via microdnf or otherwise) have a comment on why they are added
- [ ] Things not downloaded from Red Hat repositories should be mirrored in the Stackable repository and downloaded from there
- [ ] All packages should have (if available) signatures/hashes verified
- [ ] Add an entry to the CHANGELOG.md file
- [ ] Integration tests ran successfully
```

<details>
<summary>TIP: Running integration tests with a new product image</summary>

The image can be built and uploaded to the kind cluster with the following commands:

```shell
bake --product <product> --image-version <stackable-image-version>
kind load docker-image <image-tagged-with-the-major-version> --name=<name-of-your-test-cluster>
```

See the output of `bake` to retrieve the image tag for `<image-tagged-with-the-major-version>`.
</details>

### CODE REVIEW

File: stackable-base/Dockerfile [hunk 1]
```diff
@@ -29,7 +29,7 @@ RUN cd ./config-utils && \
     cargo auditable build --release && cargo cyclonedx --output-pattern package --all --output-cdx
 
 # Manifest list digest because of multi architecture builds ( https://www.redhat.com/architect/pull-container-image#:~:text=A%20manifest%20list%20exists%20to,system%20on%20a%20specific%20architecture )
-FROM registry.access.redhat.com/ubi9/ubi-minimal@sha256:a7d837b00520a32502ada85ae339e33510cdfdbc8d2ddf460cc838e12ec5fa5a AS final
+FROM registry.access.redhat.com/ubi9-minimal@sha256:a7d837b00520a32502ada85ae339e33510cdfdbc8d2ddf460cc838e12ec5fa5a AS final
```
Comment (lfrancke) [inline stackable-base/Dockerfile:32]: Why these changes?

Comment (sbernauer) [reply stackable-base/Dockerfile:32]: They where not needed, we just wanted to switch to what we *think* is the new naming scheme to be safe going forward

Comment (NickLarsenNZ) [reply stackable-base/Dockerfile:32]: Then I changed it back in https://github.com/stackabletech/docker-images/pull/807 (because that's how it appears at the linked site I have added in my PR :shrug:)

Note: @sbernauer did ask me about this and I double checked the commit hashes match.
